sql/moz-fx-data-marketing-prod/ga_derived/blogs_daily_summary_v1/init.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE IF NOT EXISTS
+  `moz-fx-data-marketing-prod.ga_derived.blogs_daily_summary_v1`(
+    date DATE,
+    deviceCategory STRING,
```

Comment:
The existing queries have a mix of camel case and underscores. I kept them as is for now so consumers don't need to change any logic but would it be easy to change those so we can standardize to underscore?

Reply:
I think this was because the GA drops have camel case and I hadn't really settled on a method. I'm fine with changing them. Once we migrate, all the end points that use the data will need to be updated so we may as well go in and make the changes now to be consistent with the way you guys do things.